miniwin/src/internal/d3drmmesh_impl.h

```diff
@@ -13,7 +13,7 @@ struct MeshGroup {
 	int vertexPerFace = 0;
 	int version = 0;
 	std::vector<D3DRMVERTEX> vertices;
-	std::vector<DWORD> indices;
+	std::vector<unsigned int> indices;
```

Comment:
correct the indices to DWORD instead:

https://github.com/isledecomp/isle-portable/blob/e6a005fd331e8daa7893a4287bb02a1123ac192b/miniwin/src/internal/d3drmmesh_impl.h#L16